src/app/application/components/app-collection/AppCollectionAction.vue

```diff
@@ -0,0 +1,8 @@
+<template>
+  <KDropdownItem />
```

Comment:
**Issue**: For this to work well with the tooling around Vue and type checking of props, props need to be passed explicitly. This is in fact already a problem with `AppCollection` where the props we pass to it aren’t any longer type-checked against the `KTable` props (e.g. you can pass `:headers="1"` and nothing will alight with errors).